cmd/controller/app/server.go

```diff
@@ -79,13 +78,6 @@ func NewControllerManagerCommand() *cobra.Command {
 		fs.AddFlagSet(f)
 	}
 
-	usageFmt := "Usage:\n  %s\n"
```

Comment:
How about keeping the help print function? And just remove the term part.

Reply:
I agree. Term package was used to get the width of the running terminal, in order to beautify the help message printed in the terminal. By setting the max width of the print message, the print function can wrap line according the width. But IMO there is no need to wrap the message. More attention should be paid to the devops core code, not to the beautiful printing help message.